src/navigation/components/groupDraftItem/GroupDraftItem.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import Icon from '../../../media/Icon';
+
+export const PROFILE_GROUP_DRAFT_LIST_ITEM_CLASS = 'profileDropdown-draft-group';
+
+export const GroupDraftItem = ({ groupDraft }) => {
+	return (
+		<li
+			key={groupDraft.urlname}
+			className={cx(PROFILE_GROUP_DRAFT_LIST_ITEM_CLASS, 'list-item')}
+		>
+			<p>{groupDraft.name}</p>
+			<div>
+				<span>{groupDraft.status}</span>
+				<a className="draftprofiledropdown" href={groupDraft.urlname}>
```

Comment:
If `groupDraft.urlname` is actually the group `urlname`, I think this will end up being something like `href='my-group-name'`, which won't always navigate to the right place. It's sort of tricky because ideally we would generate the link `href` using the user's locale (for something like `/fr-FR/my-group-name`) and we would avoid doing a full server render, but that's only possible from inside the applications with `<Link ...>`

The href should maybe be

```jsx
href={`/${groupDraft.urlname`}
```

as a compromise? The app server will perform the necessary language-aware redirects for that request

Reply:
actually will only redirect to `/create` all the time, probably to change it to draftUrlName?

Reply:
ah, ok. yeah `draftUrlname` might be a good name for it, although if it's expected to be a url path that includes a leading slash, `draftUrlpath` might be better